python/cudf/cudf/core/dataframe.py

```diff
@@ -291,12 +291,14 @@ def _getitem_tuple_arg(self, arg):
                         {tmp_col_name: column.arange(len(tmp_arg[0]))},
                         index=as_index(tmp_arg[0]),
                     )
+                    cantor_name = "_" + "_".join(columns_df._data.names)
```

Comment:
```suggestion
                    cantor_name = "_" + "_".join(map(str, columns_df._data.names))
```
Should we guard against having mixed column names like incase of `[0, 1, "a"]` this will error.

Reply:
Thanks, good catch!